static/js/vue-cdr-access/src/components/searchWrapper.vue

```diff
@@ -3,7 +3,7 @@ Top level component wrapper for search pages
 -->
 <template>
     <header-small/>
-    <div>
+    <div v-if="!show_404 && !show_503">
```

Comment:
Alternatively we could put the error pages first in the template and make the actual page the else case, but i think its fine either way

Reply:
I'm kind of inclined to leave it as it is, as they're at least tucked out of the way a bit.